boards/ble400/include/board.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2016 Stephane D'Alu
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    boards_ble400 BLE400
+ * @ingroup     boards
+ * @brief       Board specific files for the BLE400 board
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the BLE400 board
+ *
+ * @author      Stephane D'Alu <sdalu@sdalu.com>
+ */
+
+#ifndef BOARD_H_
+#define BOARD_H_
+
+#include "cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Xtimer configuration
+ * @{
+ */
+#define XTIMER                      (0)
+#define XTIMER_CHAN                 (0)
+#define XTIMER_MASK                 (0xff000000)
+#define XTIMER_SHIFT_ON_COMPARE     (2)
```

Comment:
What is not needed? The whole XTIMER section or only the XTIMER_SHIFT_ON_COMPARE line?